src/lib/jsonParse/jsonRegisterContextRequest.cpp

```diff
@@ -204,11 +204,6 @@ static std::string craMetadata(const std::string& path, const std::string& value
   LM_T(LmtParse, ("Creating a metadata"));
 
   reqDataP->rcr.attributeMetadataP = new Metadata();
-
-  reqDataP->rcr.attributeMetadataP->type  = "";
```

Comment:
This initialization was unneced. This is work for the constructor. Maybe it comes from ancient days in which constructor where much more limited...

Reply:
NTC (just informative)